web/HTML/tutorial/tutorial.md

```diff
@@ -103,37 +103,37 @@ The following elements can be modeled as gyrators,
 
 Linear graph representations of systems use "nodes" and connecting "branches" to diagram a system.
 Each node represents an independent across-variable value.
-Branches are created for each element in the system.
-In non electrical systems A type elements always connect to a ground node.
+Branches are created for each element in the system. **REVIEW: Could you explain this idea more?**
+In non-electrical systems, A type elements always connect to a ground node.
 Arrows on branches point in the direction of decreasing across-variable.
 Arrows on transformers and gyrators always point towards ground.
 An example of a linear graph appears in the Example section of this tutorial.
 
 ### Normal Trees
 
-In order to find the primary and secondary variables a normal tree can be constructed.
+In order to find the primary and secondary variables **REVIEW: You have not yet defined primary and secondary variables**, a normal tree can be constructed.
```

Comment:
These are defined in the next section. It would be difficult to define these terms without first discussing the normal tree.